mammoth_gazebo/launch/include/navigation.launch.py

```diff
@@ -85,7 +84,7 @@ def generate_launch_description():
 
         # Nodes
         slam_toolbox,
-        # nav2_stack,  # 5/23/21 dcutting133: currently this is very buggy.
+        #nav2_stack,  # 5/23/21 dcutting133: currently this is very buggy.
```

Comment:
uncomment and use the correct launch file for what is being launched by you?

Reply:
fixed in my latest commit:) I had to make a nav2 launch file from scratch for it to work